docs/content/preview/admin/yb-admin.md

```diff
@@ -1793,6 +1793,47 @@ yb-admin \
 * *replication_name*: The name of the replication to be enabled or disabled.
 * `0` | `1`: Disabled (`0`) or enabled (`1`). Default is `1`.
 
+#### get_xcluster_safe_time
+
+Reports the skew in xCluster Safe time between the laggiest tablet and the most caught up tablet. 
```

Comment:
```suggestion
Reports the skew in xCluster safe time between the laggiest tablet and the most caught up tablet. 
```

Reply:
Actually this commands behaviour has been changed a bit, so this description is no longer quite accurate.

Should now say:
```
Reports the current xCluster safe time for each namespace, which is the time at which reads will be performed.
```

Reply:
Added in https://github.com/yugabyte/yugabyte-db/pull/17327/commits/e5ea855a805e34b499f6ad964969d5c58519f545